### PR TITLE
data: ADITYADAS1999

### PR DESCRIPTION
**Pull request by ADITYADAS1999**
## Check List (Check all the applicable boxes) 

- [✅] My code follows the code style of this project.
- [✅] My change requires changes to the documentation.
- [✅] I have updated the documentation accordingly.
- [✅] All new and existing tests passed.
- [✅] This PR does not contain plagiarized content.
- [✅] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2745"></a>

### CODE REVIEW

File: data/ADITYADAS1999.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {
+      "name": "visit my Github profile",
+      "url": "https://adityadas1999.github.io/",
+      "icon": "FaGithub"
+    }
+  ]
+}
+
+
+
```
Comment (Panquesito7) [inline data/ADITYADAS1999.json:37]: There are three extra lines in here.
```suggestion
```

Comment (ADITYADAS1999) [reply data/ADITYADAS1999.json]: > There are three extra lines in here.

Thanks for reviewing the PR [Panquesito7](https://github.com/Panquesito7) 😃

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/ADITYADAS1999.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "icon": "FaGlobe"
+    },
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://www.youtube.com/channel/UCUBqrPu1GoR0Cfgl5rDHIbg",
+      "icon": "FaYoutube"
+    },
+    {
+      "name": "visit my Github profile",
+      "url": "https://adityadas1999.github.io/",
```
Comment (Pradumnasaraf) [inline data/ADITYADAS1999.json:30]:
```suggestion
      "url": "https://github.com/ADITYADAS1999",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

It looks like you have mistakenly put the portfolio link  instead of GitHub link

**Comment by ADITYADAS1999** [comment]
> Hey,
> 
> It looks like you have mistakenly put the portfolio link instead of GitHub link

ohh no, I will change this👀

File: data/ADITYADAS1999.json (hunk 1 above)
Comment (eddiejaoude) [reply data/ADITYADAS1999.json]: We have Prettier GitHub Action that should fix these formatting issues (if the json is valid), so not a blocker

**Review by eddiejaoude — APPROVED** [review]
Thanks for updating 👍  Great collaboration everyone 🎉

**Pull request merged**